decidim-results/app/views/decidim/results/results/show.html.erb

```diff
@@ -34,6 +37,7 @@
         </div>
       </div>
     </div>
+    <%= render partial: "decidim/shared/share_modal" %>
```

Comment:
`content_for` is not used inside this and should not be attached to it anyway. Meta tags are and can be used in many other things, not only in the share modal.